src/Gang/Gang.ts

```diff
@@ -325,7 +325,14 @@ export class Gang {
     if (this.members.length < numFreeMembers && this.respect < Math.pow(recruitCostBase, numFreeMembers)) {
```

Comment:
There's actually an issue here too: If respect is 10 (for instance), that is < 5**3, so this line will trigger. However, the correct result is `1 + numFreeMembers - this.members.length`, i.e. it should go to the next part.

The real condition here should be `< recruitCostBase`.

Reply:
Based on my changes to the next check, I just completely removed this statement.  I'm already account for the free members in the next check by default.  It also fixes the bug where the available count would be off if you didn't recruit the first three members before gaining enough respect for the next member.

Reply:
There is one case that was covering that is still important, or you need to upgrade the `max` to a `clamp`: When respect is 0, the log becomes -Infinity, leading to -Infinity recruitable members currently.

Reply:
It looks like you start with 1 respect.  I'm not sure if it's even possible to lose respect as I thought when a gang member dies/ascends you just lose the respect they've gained.  Regardless, I believe I followed what you were suggesting and added the check to the appropriate place.